membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -93,8 +89,8 @@ object AccountDetails {
       })
 
       val sortedPlans = subscription.plans.list.sortBy(_.start.toDate)
-      val currentPlans = sortedPlans.filter(planIsCurrent)
-      val futurePlans = sortedPlans.filter(_.start.isAfter(now))
+      val currentPlans = sortedPlans.filter(plan => plan.start.isBefore(now.plusDays(1)) && plan.end.isAfter(now))
```

Comment:
maybe it might be more clear to say 
```scala
      val currentPlans = sortedPlans.filter(plan => !plan.start.isAfter(now) && plan.end.isAfter(now))

```
assuming that is what we are trying to do

Reply:
too late

Reply:
great suggestion, unfortunately just merged the PR. i'll do it in a follow up

Reply:
https://github.com/guardian/members-data-api/pull/375